### PR TITLE
docs(readme): update usage instructions for OpenEBS 1.1.0

### PR DESCRIPTION
**Pull request by kmova**
Refactored the usage instructions to provide few additional
details and verification steps.

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,179 @@
...
+   registered with Kubernetes for receiving any requests related to
+   `openebs-csi.openebs.io`  
+
+3. OpenEBS CSI Controller will create a custom resource called 
+   CStorVolumeClaim(CVC) and returns the details of the newly 
+   created object back to Kubernetes. The cStorVolumeClaim will be
+   monitored by the cstor-operator (embedded in m-apiserver). The
+   cstor-operator will process the request once the Kubernetes 
+   determines the node on which application using the PVC should 
+   be scheduled.
```
Comment (prateekpandey14) [inline README.md]: The cstor-operator will wait for process the request until the application pods using the PVC has been scheduled on a node ?

File: README.md [hunk 2]
```diff
@@ -1,105 +1,179 @@
...
+
+1. Create PVC with Storage Class referring to OpenEBS CSI Driver.
+
+2. Kubernetes will pass the PV creation request to the OpenEBS
+   CSI Controller service via `CreateVolume()`, as this controller
+   registered with Kubernetes for receiving any requests related to
+   `openebs-csi.openebs.io`  
+
+3. OpenEBS CSI Controller will create a custom resource called 
+   CStorVolumeClaim(CVC) and returns the details of the newly
```
[the remark this answers is not in the document]
Comment (kmova) [reply README.md]: done. thank you

File: README.md [hunk 3]
```diff
@@ -1,105 +1,179 @@
...
+
+2. Create a Storage Class to dynamically provision volumes 
+   using OpenEBS CSI provisioner. A sample storage class looks like:
+   ```
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1
+   metadata:
+     name: openebs-csi-cstor-sparse
+     namespace: kube-system
+     annotations:
```
[the remark this answers is not in the document]
Comment (kmova) [reply README.md]: done. thank you!

File: README.md (hunk 1 above)
Comment (kmova) [reply README.md]: reworded a bit. please check.

File: README.md [hunk 4]
```diff
@@ -1,105 +1,178 @@
...
+   ```
+
+2. Create a Storage Class to dynamically provision volumes 
+   using OpenEBS CSI provisioner. A sample storage class looks like:
+   ```
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1
+   metadata:
+     name: openebs-csi-cstor-sparse
+     namespace: kube-system
```
Comment (prateekpandey14) [inline README.md]: ahh missed to add this comment earlier, SC is cluster scoped resources , so namespace not needed.

File: deploy/sc.yaml [hunk 5]
```diff
@@ -1,11 +1,10 @@
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
-  name: openebs-csi-default
+  name: openebs-csi-cstor-sparse
   namespace: kube-system
```
Comment (prateekpandey14) [inline deploy/sc.yaml]: Same comment as above

Comment (kmova) [reply deploy/sc.yaml]: good one. resolved both.

**Pull request merged**